DataProcessing/datax-utility/pom.xml

```diff
@@ -125,7 +125,7 @@ SOFTWARE
         <dependency>
             <groupId>com.microsoft.azure</groupId>
             <artifactId>applicationinsights-core</artifactId>
-            <version>2.6.4</version>
+            <version>3.4.1</version>
```

Comment:
Changing the version here involves changing the version in the spark job configs